projects/steering/src/steering_control_stalk.c

```diff
@@ -0,0 +1,71 @@
+#include "steering_control_stalk.h"
+#include <stdio.h>
+#include "adc.h"
+#include "adc_periodic_reader.h"
+#include "can.h"
+#include "can_ack.h"
+#include "can_fifo.h"
+#include "can_fsm.h"
+#include "can_hw.h"
+#include "can_msg.h"
+#include "can_rx.h"
+#include "can_transmit.h"
+#include "event_queue.h"
+#include "exported_enums.h"
+#include "fsm.h"
+#include "gpio.h"
+#include "interrupt.h"
+#include "log.h"
+#include "soft_timer.h"
+#include "status.h"
+#include "steering_can.h"
+#include "steering_digital_input.h"
+#include "steering_events.h"
+
+// Will be edited later
+#define STEERING_CONTROL_STALK_LEFT_SIGNAL_VOLTAGE 1000

```

Comment:
might wanna mention units somewhere for someone looking at this later. Also what's are these not the final voltages?

Reply:
Yeah, I haven't got a chance to check the actual voltages from the control stalk